### PR TITLE
Decrease bound check PVC size to 10Mi

### PR DESCRIPTION
**Pull request by arnongilboa**
We would like to check provisioners can create and bound a 10Mi PVC, as the kubevirt requirement for the "Backend state PVC" is 10Mi.

### CODE REVIEW

File: pkg/internal/checkup/checkup.go [hunk 1]
```diff
@@ -456,6 +456,7 @@ func (c *Checkup) waitForPVCBound(ctx context.Context, result, errStr *string) {
 
 	log.Printf("Waiting for PVC %q bound", pvcName)
 	if err := wait.PollImmediateWithContext(ctx, pollInterval, time.Minute, conditionFn); err != nil {
+		log.Printf("PVC %q failed to bound", pvcName)
```
Comment (ShellyKa13) [inline pkg/internal/checkup/checkup.go:459]: Do we explain in anyway why the PVC bound failed?

Comment (arnongilboa) [reply pkg/internal/checkup/checkup.go:459]: Usually in this case the PVC is stuck in Pending phase without any status Conditions updated, so it simply timeouts waiting for Bound. It usually means the provisioner does not support such small PVC.

Comment (ShellyKa13) [reply pkg/internal/checkup/checkup.go:459]: yes sure, though there can be other reasons why PVC failed to bound but I see it is difficult to know this in the test. thanks

**Review by ShellyKa13 — APPROVED** [review]
/lgtm

**Pull request merged**